github3/gists/file.py

```diff
@@ -31,7 +31,16 @@ def _update_attributes(self, attributes):
         #: The size of the file.
         self.size = attributes.get('size')
         #: The content of the file.
-        self.content = attributes.get('content')
+        self.original_content = attributes.get('content')
 
     def _repr(self):
         return '<Gist File [{0}]>'.format(self.name)
+
+    def content(self):
+        """Retrieve contents of file from key 'raw_url' if there is no
+        'content' key in Gist object.
+        """
+        resp = self._get(self.raw_url)
+        if self._boolean(resp, 200, 404):
```

Comment:
More like "use them if you want, don't use them if you don't". So long as you don't nest ternaries or make them long enough to be multiline, I'm fine with them. I'm not strongly for or against them.